README.md

```diff
@@ -21,6 +21,16 @@ Make sure your `$HOME/.composer/vendor/bin` directory is in your PATH (or the eq
 Then you can run this script with `upgrade-code <command>` in your project root. If not running in the root,
 use --root-dir=/path.
 
+### Install as a PHAR executable
+
+The upgrader is also available as a phar executable. This can be helpful if installing the upgrader globally through composer conflicts with other libraries.
+
+To install the PHAR executable:
+
+1. [Download the upgrader as a PHAR executable](releases/download/1.2.0/upgrade-code.phar) or `wget https://github.com/silverstripe/silverstripe-upgrader/releases/download/1.2.0/upgrade-code.phar`
+1. Make the file executable `chmod +x upgrade-code.phar`
```

Comment:
I do quite enjoy this abuse of markdown, however markdown is meant to be a format that's easily readable both in it's rendered and raw format. I fear we're prioritising clean diffs over readability of the raw file here.

Reply:
I presume you're referring to the repetitive use `1. ` instead of having an incremented number for each step?

Given it's a small list, I'm happy to use `1.`, `2.`, `3.`, etc.